journald/journald.hpp

```diff
@@ -13,12 +13,17 @@
 #include <stout/option.hpp>
 #include <stout/protobuf.hpp>
 
+#include <stout/os/pagesize.hpp>
+#include <stout/os/shell.hpp>
+
 
 namespace mesos {
 namespace journald {
 namespace logger {
 
 const std::string NAME = "mesos-journald-logger";
+const std::string CONF_SUFFIX = ".logrotate.conf";
+const std::string STATE_SUFFIX = ".logrotate.state";
```

Comment:
LOGROTATE_CONF_SUFFIX and LOGROTATE_STATE_SUFFIX, since these are only meaningful if your logger destination type includes logrotated sandbox